spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectionFactory.scala

```diff
@@ -73,9 +96,36 @@ object DefaultConnectionFactory extends CassandraConnectionFactory {
       .withSocketOptions(options)
   }
 
+  /** Returns the Cluster.Builder object used to setup Cluster instance. */
+  def clusterSSLBuilder(conf: CassandraConnectorConf, builder: Cluster.Builder): Cluster.Builder = conf.sslEnabled match {
+    case false => builder
+    case true => clusterSSLBuilderOptions(conf, builder)
+  }
+
+  private def clusterSSLBuilderOptions(conf: CassandraConnectorConf, builder: Cluster.Builder): Cluster.Builder = conf.sslTrustStorePath match {
+    case None => builder.withSSL()
+    case path: Some[String] => {
+      val trustStoreFile = new FileInputStream(path.get)
+      val context = SSLContext.getInstance("SSL")
```

Comment:
I'm for adding all the configuration options which Cassandra allow you to configure. Otherwise, soon we may have another ticket from some user how needs to configure it differently. This wouldn't add much complexity, would it?